library/src/test/java/com/datatorrent/lib/streamquery/AntiJoinOperatorTest.java

```diff
@@ -0,0 +1,120 @@
+/**
+ * Copyright (C) 2015 DataTorrent, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datatorrent.lib.streamquery;
+
+import com.datatorrent.lib.streamquery.condition.Condition;
+import com.datatorrent.lib.streamquery.condition.JoinColumnEqualCondition;
+import com.datatorrent.lib.streamquery.index.ColumnIndex;
+import com.datatorrent.lib.testbench.CollectorTestSink;
+
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.HashMap;
+
+/**
+ *
+ * Functional test for {@link AntiJoinOperator }.
+ *
+ */
+public class AntiJoinOperatorTest
+{
+  @SuppressWarnings({"rawtypes", "unchecked"})
+  @Test
+  public void testSqlSelect()
+  {
+    Logger LOG = LoggerFactory.getLogger(AntiJoinOperator.class);
+
+    // create operator
+    AntiJoinOperator oper = new AntiJoinOperator();
+    CollectorTestSink sink = new CollectorTestSink();
+    oper.outport.setSink(sink);
```

Comment:
TestUtils.setSink(...)  can be used here. This can avoid specifying "rawtypes", "unchecked" warnings.